### PR TITLE
Windows builds

### PR DESCRIPTION
**Pull request by vswarte**
Goal of the PR is to get builds possible for windows

### CODE REVIEW

File: crates/dvdbnd/src/lib.rs [hunk 1]
```diff
@@ -140,6 +143,7 @@ impl DvdBnd {
                     data_cipher.decrypt_blocks(blocks);
                 }
 
+                #[cfg(not(target_family = "windows"))]
```
Comment (garyttierney) [inline crates/dvdbnd/src/lib.rs:146]:
```suggestion
                #[cfg(not(target_family = "unix"))]
```

Comment (vswarte) [reply crates/dvdbnd/src/lib.rs:146]: Uh I don't think that's correct?
Although its probably better to do `#[cfg(target_family = "unix")]` as it doesn't seem to be support at all outside of unix

Comment (garyttierney) [reply crates/dvdbnd/src/lib.rs:146]: > Uh I don't think that's correct?

Based on what? https://github.com/RazrFalcon/memmap2-rs/blob/master/src/lib.rs#L47-L48

Comment (vswarte) [reply crates/dvdbnd/src/lib.rs:146]: Think I was staring at a cached version of your suggestion

**Review by garyttierney — APPROVED** [review]

File: crates/dvdbnd/src/lib.rs (hunk 1 above)
Comment (vswarte) [reply crates/dvdbnd/src/lib.rs]: febdc3318e068f2d5fd365c745237814c50f783c should do the trick

**Pull request closed without merging**